server.lua

```diff
@@ -1,7 +1,21 @@
 -- CONFIG --
 roleList = Config.RoleList;
+inheritances = Config.Inheritances
 
 -- CODE --
+Citizen.CreateThread(function()
+	--checks to see if players are online on resource start. if so it registers them.
+	if GetNumPlayerIndices() ~= 0 then
+		for _, ID in pairs(GetPlayers()) do
+			print("test")
```

Comment:
Should not be having a random print statement here

Reply:
> Should not be having a random print statement here

I removed that on the 2nd commit.